resources/views/platform/components/modal-content/beaten-game-credit.blade.php

```diff
@@ -0,0 +1,75 @@
+@props([
+    'gameTitle' => '',
+    'progressionAchievements' => [],
+    'winConditionAchievements' => [],
+    'unlockContext' => 's:|h:',
+])
+
+<?php
+[$softcoreUnlocks, $hardcoreUnlocks] = explode("|", $unlockContext);
+$softcoreIds = explode(",", substr($softcoreUnlocks, 2)); // skip "s:"
+$hardcoreIds = explode(",", substr($hardcoreUnlocks, 2)); // skip "h:"
```

Comment:
When a Progression achievement is demoted, the dialog can be opened from the Unofficial list. It only shows the remaining core achievements, and none of them appear unlocked.

The dialog should probably just be disabled when viewing Unofficial achievements.

Reply:
Agreed, disabling viewing it on Unofficial is probably the best experience. Resolved in https://github.com/RetroAchievements/RAWeb/pull/1810/commits/af3bbabc7dc3ba6bfd17ce03935de7829887be14.